src/Files.App/Views/PaneHolderPage.xaml

```diff
@@ -56,6 +42,7 @@
 				x:Name="LeftColumn"
 				Width="*"
 				MinWidth="300" />
+			<ColumnDefinition x:Name="SizerColumnDefinition" Width="Auto" />
```

Comment:
I'm not sure if it's related to this exact change, but there seems to be some issue with the visual states.
![image](https://github.com/files-community/Files/assets/39923744/48dc581b-4ec9-4cf5-b99b-ffa2e49be521)

Reply:
Wdym? Overwrapping?

Reply:
Yeah, it seems to overlap a couple pixels on the right pane.

Reply:
This is existing issue. You can see it in stable too

Reply:
I can't repro this in stable

Reply:
I guess we can solve in another PR.

Reply:
In fact, I can repro this with a single pane as well.

Reply:
You're correct! However, it seems to be more noticable with these changes so I'll push a fix.